src/Security/Member.php

```diff
@@ -57,6 +55,9 @@
  * @property int $FailedLoginCount
  * @property string $DateFormat
  * @property string $TimeFormat
+ * @method SilverStripe\ORM\ManyManyList<Group> Groups()
```

Comment:
```suggestion
 * @method ManyManyList<Group> Groups()
```
And in a few other classes as well - the class is imported but we're still using the FQCN

Reply:
Whoops, forgot to handle belongs_many_many in the code writer